main.go

```diff
@@ -172,3 +179,26 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+// getValues returns the stagingRepoUrl unless the appGroup controller
+// is run in a debug mode, then it returns the port forwarded url
+func getValues(stagingHelmURL, tempChartDir, mode string) (string, string, string) {
+	switch mode {
+	case "debug":
+		return "http://127.0.0.1:8080", "http://orkestra-chartmuseum.orkestra:8080", getLocalTempChartDir()
+	default:
+		return stagingHelmURL, stagingHelmURL, tempChartDir
+	}
+}
+
+func getLocalTempChartDir() string {
```

Comment:
It might be a better choice to use https://golang.org/pkg/os/#TempDir instead of creating a local dir.